### ISSUE TITLE
Extending an extended network

### ISSUE THREAD

**Issue by susancoort** [CyTargetLinker/cytargetlinker.app#7]
When extending an extended network the clone is not automatically created. Is this what you still want to implement? So the option to created a clone or not?

**Comment by fehrhart** [comment]
Same here, extending an extended network (after saving, closing cytoscape) does not recognise previous extensions anymore.
(see screenshot all step 1 labelled in CTL_EXT)

![grafik](https://user-images.githubusercontent.com/11389574/36212509-693d2cec-11a4-11e8-9ae0-a723cc2bbc26.png)

**Comment by mkutmon** [comment]
Currently, I only implemented the cloning of the initial network. After that all extensions are done in the same network. In the future, there will be an option if the network should be cloned or not - but that will be available in version 4.1.0. The same is true for session loading (although I might implement this tonight because I think it's too important - even though the older version didn't have it).

**Comment by mkutmon** [comment]
see #1 and #3

**Comment by fehrhart** [comment]
yes, but I remember you showed me it will count up from number of extensions (step 1, 2, 3) - that does not work if cytoscape is closed and reopened again

**Comment by mkutmon** [comment]
yes! it only works if you extend it within the same session. if you reopen the session information is not loaded and cytargetlinker therefore doesn't have the details about the extensions yet.
